content/es/what-is-opentelemetry.md

```diff
@@ -0,0 +1,132 @@
+---
+title: ¿Qué es OpenTelemetry?
+description: Qué es y qué no es OpenTelemetry, una breve explicación
+weight: 150
+default_lang_commit: 13c2d415e935fac3014344e67c6c61556779fd6f
+cSpell:ignore: microservicios extensibilidad
+---
+
+OpenTelemetry es:
+
+- Un framework de
+  [observabilidad](/docs/concepts/observability-primer/#what-is-observability) y
+  un conjunto de herramientas diseñado para crear y gestionar datos de
+  telemetría tales como [trazas](/docs/concepts/signals/traces/),
+  [métricas](/docs/concepts/signals/metrics/) y
+  [logs](/docs/concepts/signals/logs/).
+- Independiente de proveedores y herramientas, lo que significa que puede usarse
+  con una amplia variedad de backends de observabilidad, incluyendo
+  [Jaeger](https://www.jaegertracing.io/) y
+  [Prometheus](https://prometheus.io/), así como soluciones comerciales.
+- No es un backend de observabilidad como Jaeger, Prometheus, o productos
+  comerciales de terceros.
+- Enfocado en la generación, recopilación, gestión y exportación de telemetría.
+  Uno de los objetivos principales de OpenTelemetry es que puedas instrumentar
+  fácilmente tus aplicaciones o sistemas, sin importar el lenguaje de
+  programación, infraestructura o entorno de ejecución. El almacenamiento y
+  visualización de la telemetría se deja intencionalmente a otras herramientas.
```

Comment:
I think this should be "intencionadamente". Fundeu speaks a bit more about this here: https://www.fundeu.es/recomendacion/intencionalidad-intencion/
```suggestion
  visualización de la telemetría se deja intencionadamente a otras herramientas.
```

Reply:
Good point!